core/src/main/java/org/mule/runtime/core/api/streaming/bytes/FileStoreCursorStreamConfig.java

```diff
@@ -4,7 +4,7 @@
  * license, a copy of which has been included with this distribution in the
  * LICENSE.txt file.
  */
-package org.mule.runtime.core.internal.streaming.bytes;
```

Comment:
don't see a reason for that. What's your concern?

Reply:
I see that the other FileStoreConfig that is created in this PR has it, and assumed that this one did not have it because it was internal. I do not have a specific concern.